mcs/class/System.Data/corefx/RuntimeInformation.cs

```diff
@@ -0,0 +1,29 @@
+using System;
```

Comment:
I still don't like this copy, if we you really need it we need to split Facades build so you can take dependency on Facades/System.Runtime.InteropServices.RuntimeInformation